libs/coin-evm/docs/specs.md

```diff
@@ -0,0 +1,15 @@
+# `specs.ts`
```

Comment:
there is already doc in the code let's merge infos

Reply:
Same answer as above